### PR TITLE
AC: support DICOM reading

### PR DESCRIPTION
**Pull request by eaidova**

### CODE REVIEW

File: tools/accuracy_checker/accuracy_checker/data_readers/data_reader.py [hunk 1]
```diff
@@ -436,3 +442,18 @@ def read(self, data_id):
 
     def read_item(self, data_id):
         return DataRepresentation(*self.read_dispatcher(data_id), identifier=data_id)
+
+
+class DicomReader(BaseReader):
+    __provider__ = 'dicom_reader'
```
Comment (eizamaliev) [inline tools/accuracy_checker/accuracy_checker/data_readers/data_reader.py:448]: Don't forget add it to docs!

**Comment by eaidova** [comment]
Jenkins please retry a build

File: tools/accuracy_checker/accuracy_checker/data_readers/data_reader.py (hunk 1 above)
Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/data_readers/data_reader.py:448]: and CI requirements :)

Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/data_readers/data_reader.py:448]: fixed, please take a look again

File: tools/accuracy_checker/accuracy_checker/annotation_converters/README.md [hunk 2]
```diff
@@ -140,7 +140,7 @@ Accuracy Checker supports following list of annotation converters and specific f
   * `data_dir` - path to folder, where images in low and high resolution are located.
   * `lr_suffix` - low resolution file name's suffix (default lr).
   * `hr_suffix` - high resolution file name's suffix (default hr).
-  * `annotation_loader` - which library will be used for ground truth image reading. Supported: `opencv`, `pillow` (Optional. Default value is pillow). Note, color space of image depends on loader (OpenCV uses BGR, Pillow uses RGB for image reading).
+  * `annotation_loader` - which library will be used for ground truth image reading. Supported: `opencv`, `pillow`, `dicon`. (Optional. Default value is pillow). Note, color space of image depends on loader (OpenCV uses BGR, Pillow uses RGB for image reading).
```
Comment (eizamaliev) [inline tools/accuracy_checker/accuracy_checker/annotation_converters/README.md:143]:
```suggestion
  * `annotation_loader` - which library will be used for ground truth image reading. Supported: `opencv`, `pillow`, `dicom`. (Optional. Default value is pillow). Note, color space of image depends on loader (OpenCV uses BGR, Pillow uses RGB for image reading).
```

File: tools/accuracy_checker/accuracy_checker/adapters/image_processing.py [hunk 3]
```diff
@@ -49,7 +49,10 @@ def parameters(cls):
                 description='The value on which prediction pixels should be multiplied for scaling to range '
                             '[0, 255] (usually it is the same scale (std) used in preprocessing step))'
             ),
-            'target_out': StringField(optional=True, description='Target super resolution model output')
+            'target_out': StringField(optional=True, description='Target super resolution model output'),
+            "cast_to_uint8": BoolField(
+                optional=True, default=True, description="Cast output image to [0, 255] diapasone"
```
Comment (eizamaliev) [inline tools/accuracy_checker/accuracy_checker/adapters/image_processing.py:54]:
```suggestion
                optional=True, default=True, description="Cast prediction values to integer within [0, 255] range"
```

**Pull request merged**